app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">JavaBaseProject</string>
+    <string name="text_button_main_activity">button</string>
+    <string name="text_view_main_activity">text</string>
 </resources>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done